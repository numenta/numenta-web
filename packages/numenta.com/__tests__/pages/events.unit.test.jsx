import React from 'react'
import renderer from 'react-test-renderer'
import stubContext from 'react-stub-context'

import context from '__tests__/__mocks__/reactContextMock'

import EventsPage from '../../pages/events'


describe('Events Page React component', () => {

  it('Renders correctly', () => {
    const EventsPageStubbed = stubContext(EventsPage, context)
    const component = renderer.create(
      <EventsPageStubbed />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

})
