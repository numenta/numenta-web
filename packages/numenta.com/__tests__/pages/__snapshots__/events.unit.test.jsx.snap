exports[`Events Page React component Renders correctly 1`] = `
<article
  className="events">
  <section
    className="section"
    id={undefined}>
    <h1
      className="headline">
      Events
    </h1>
    <div
      className="article">
      <span
        className="anchor"
        id="upcoming" />
      <h2
        className="h2">
        Upcoming Events
      </h2>
      <ul
        className="ul" />
      <span
        className="anchor"
        id="past" />
      <h2
        className="h2">
        Past Events
      </h2>
      <ul
        className="ul" />
    </div>
  </section>
</article>
`;
